### PR TITLE
Added ARM64 build into CI

### PR DESCRIPTION
**Pull request by rafaelsamenezes**
This PR will add build checks for ubuntu ARM64 systems.

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -135,7 +143,7 @@ usage() {
 }
 
 # Setup build flags (release, debug, sanitizer, ...)
-while getopts hb:s:e:r:dS:c:C flag
+while getopts hb:s:e:r:dS:c:C:a flag
```
Comment (fbrausse) [inline scripts/build.sh]: Remove colon after `C`, it doesn't take an argument.

Comment (rafaelsamenezes) [reply scripts/build.sh]: Oops, removed the colon.

File: scripts/build.sh [hunk 2]
```diff
@@ -61,6 +65,15 @@ ubuntu_setup () {
     else
         echo "Configuring static Ubuntu build"
     fi
+
+    if [ $ARCH = "aarch64" ]
+    then
+	echo "Detected ARM64 Linux!"
+	# TODO: We should start using container builds in actions!
```
Comment (fbrausse) [inline scripts/build.sh:72]: What are "container builds"?

**Review by fbrausse — APPROVED** [review]
LGTM.

File: scripts/build.sh (hunk 2 above)
Comment (rafaelsamenezes) [reply scripts/build.sh:72]: Just makes the host invoke a [docker container](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container).

**Review by lucasccordeiro — APPROVED** [review]
Thanks for submitting this PR, @rafaelsamenezes.

**Pull request merged**